handsontable/src/plugins/dropdownMenu/dropdownMenu.js

```diff
@@ -234,12 +235,21 @@ export class DropdownMenu extends BasePlugin {
    * Disables the plugin functionality for this Handsontable instance.
    */
   disablePlugin() {
+    const clearColHeader = (column, TH) => updateAttributes(
+      TH, [A11Y_LABEL()[0], A11Y_HASPOPUP()[0]]
+    );
+
     this.close();
 
     if (this.menu) {
       this.menu.destroy();
     }
 
+    this.hot.addHook('afterGetColHeader', clearColHeader);
+    this.hot.addHookOnce('afterViewRender', () => {
+      this.hot.removeHook('afterGetColHeader', clearColHeader);
+    });
+
```

Comment:
I think that this kind of thing should be handled in the Walkontable renderer. Like [here](https://github.com/handsontable/handsontable/blob/develop/handsontable/src/3rdparty/walkontable/src/renderer/cells.js#L104) but for headers.